src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -118,7 +117,7 @@ public void syncDatabase() {
     /**
      * Installs the mentioned package into the local repository.
      *
-     * @param target the package to be installed
+     * @param target   the package to be installed
```

Comment:
see question above

If so: Why is it three whitespaces here and only two above?
If not:
```suggestion
     * @param target the package to be installed
```

Alternatively: would a colon, comma or semicolon be an option or is JavaDoc having issues with that?

Reply:
likewise